specification/eventgrid/data-plane/Microsoft.SignalRService/stable/2018-01-01/SignalRService.json

```diff
@@ -16,7 +16,7 @@
           "format": "date-time",
           "type": "string"
         },
-        "hub": {
+        "hubName": {
```

Comment:
It is a breaking change in stable version. We suggest introducing new api-version when including breaking changes

Reply:
While the overall EventGrid service is GA, many first party publishers onboard to EventGrid on a continuous basis to publish events to EventGrid and these events preview/GA per the timeframe of the publishing service. I don't think SignalRService events (in this swagger) have GA'ed yet, hence this change wouldn't break any customers (@zackliu, please correct me if that's not correct).

Reply:
Yes, event grid for SignalR service haven't been in public preview now. So, I think it's not a breaking change.